guizero/base.py

```diff
@@ -286,6 +286,37 @@ def children(self):
         """
         return self._children
 
+    def add_tk_widget(self, tk_widget, grid=None, align=None, visible=True, enabled=None, width=None, height=None):
+        """
+        Adds a tk widget into a guizero container.
+
+        :param tk tk_widget:
```

Comment:
Why is this parameter called tk_widget?

Reply:
Aha, the docstring is wrong. It *should* be called tk_widget but the docstring needs changing.

Reply:
Updated doc string.